src/lxcfs.c

```diff
@@ -1032,6 +1059,7 @@ static void *lxcfs_init(struct fuse_conn_info *conn)
 
 #if HAVE_FUSE3
 	cfg->direct_io = 1;
+	cfg->nullpath_ok = 0;
```

Comment:
it makes no sense to add this here, as we discussed before, `cfg` is always initialized by zeroes. If you want to add this just to make the code more explicit, then we need to explicitly initialize all other options from `cfg` structure, IMHO.

Reply:
Dropped this change, agree its unnecessary to implies a cause where none exists.